src/library/tools/pluginServer/index.ts

```diff
@@ -72,6 +72,11 @@ const main = pipe(
                   ),
                   TE.chain((body: ValidateOutputData) =>
                     validateOutput(activePlugins)(body.plugin, body.output)
+                  ),
+                  TE.chain((val) =>
+                    val.__validity == 'valid'
```

Comment:
so why not:  `val.isValid ? right : left (val.reason)`

Reply:
make sense, fixed.